scripts/update-manifest.ps1

```diff
@@ -12,7 +12,7 @@ $ManifestPath = "$ProjectPath\Package.appxmanifest"
 $xmlDoc.Package.Identity.Name="18496Starpine.Screenbox"
 $xmlDoc.Package.Identity.Publisher="CN=ABCDF790-DBE4-48F7-8204-32FCB69ADF9C"
 $xmlDoc.Package.Properties.DisplayName="Screenbox"
-$xmlDoc.Package.Applications.Application.VisualElements.DisplayName="Screenbox"
+$xmlDoc.Package.Applications.Application.VisualElements.DisplayName="ms-resource:ManifestResources/AppDisplayName"
```

Comment:
Does AppDisplayName need to be localized?

Reply:
not right now, but future proofing for other alphabets